base/docker/DockerfilePacked

```diff
@@ -0,0 +1,29 @@
+# --------------------------------------------------------------------
+# Copyright (c) 2024, WSO2 LLC. (http://wso2.com).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------
+
+# Ballerina version.
+ARG BALLERINA_VERSION
+
+FROM ballerina/ballerina:${BALLERINA_VERSION}
+
+USER root
+
+RUN bal pull ballerinax/postgresql \
```

Comment:
Shall we add SQL drivers as well?

Reply:
bal pull will not pull the latest version of dependencies. We should add null imports in a ball file and run a ball build to get the latest compatible versions of dependencies.

Reply:
Importing the null import also internally calls the bal pull is not it?

Reply:
Updated https://github.com/ballerina-platform/module-ballerina-docker/pull/1051/commits/26106567630adc08e9b042f98e18d18014aece2a